docs/quickstart.md

```diff
@@ -214,86 +204,48 @@ kubectl get clusterpackageinstall -A
 and wait for them to be `Ready:True`
 ```
 NAMESPACE           NAME               READY   SOURCE                PACKAGE
-crossplane-system   provider-aws       False   registry.upbound.io   crossplane/provider-aws:v0.9.0
-crossplane-system   stack-aws-sample   False   registry.upbound.io   crossplane/stack-aws-sample:v0.5.0
+crossplane-system   provider-aws       False   registry.upbound.io   crossplane/provider-aws:v0.10.0
+crossplane-system   stack-aws-sample   False   registry.upbound.io   crossplane/stack-aws-sample:v0.6.0
 ```
 
-#### Create aws-secret.conf
-Run the following to get your AWS Provider credentials. This requires the `aws` CLI tool to be installed and configured.
+#### Get GCP Account Keyfile
+Using an AWS account with permissions to manage AKS, RDS, networking, and identity resources in [stack-aws-sample](https://github.com/crossplane/stack-aws-sample/tree/master/kustomize/aws):
```

Comment:
It'd be nice if we listed IAM policies needed like GCP. I'll do that in a follow-up PR.